lib/jekyll-feed/feed.xml

```diff
@@ -115,7 +115,16 @@
         <summary type="html"><![CDATA[{{ post_summary | strip_html | normalize_whitespace }}]]></summary>
       {% endif %}
 
-      {% assign post_image = post.image.path | default: post.image %}
+      {% assign thumb_key = site.data.thumb_key %}
+      {% capture key_parts %}{{ thumb_key }}{% endcapture %}
```

Comment:
The key is already a string. Is the capture necessary? Couldn't it be split directly? I believe this is allowing the key to have liquid of its own which might be overkill.